cluster/gce/util.sh

```diff
@@ -480,6 +480,10 @@ function create-node-template() {
   if [[ "${PREEMPTIBLE_NODE}" == "true" ]]; then
     preemptible_minions="--preemptible --maintenance-policy TERMINATE"
   fi
+  local local_ssds=""
+  for i in ${NODE_LOCAL_SSDS}; do
```

Comment:
We don't do that for most other variables in this file. Why do you think this one is special?